Source/Core/AttributeCompression.js

```diff
@@ -131,6 +159,33 @@ define([
         return AttributeCompression.octDecodeInRange(x, y, 255, result);
     };
 
+    /**
+     * Decodes a unit-length vector in 4 byte 'oct' encoding to a normalized 3-component vector.
+     *
+     * @param {Number} x The x component of the oct-encoded unit length vector.
+     * @param {Number} y The y component of the oct-encoded unit length vector.
+     * @param {Number} z The z component of the oct-encoded unit length vector.
+     * @param {Number} w The w component of the oct-encoded unit length vector.
+     * @param {Cartesian3} result The decoded and normalized vector.
+     * @returns {Cartesian3} The decoded and normalized vector.
+     *
+     * @exception {DeveloperError} x, y, z, and w must be unsigned normalized integers between 0 and 255.
+     *
+     * @see AttributeCompression.octDecodeInRange
+     * @see AttributeCompression.octEncodeToCartesian4
+     */
+    AttributeCompression.octDecodeFromCartesian4 = function(x, y, z, w, result) {
```

Comment:
I think a `Cartesian4` argument would be cleaner. Since this is based on `octDecode`, I wonder if that should take a `Cartesian2` instead of `x` and `y`.

Reply:
> I wonder if that should take a Cartesian2 instead of x and y.

It looks like majority of the (few) use cases for `octDecode` do something to the components separately, so I'm in favor of leaving that as-is.

Having `octDecodeFromCartesian4` take `Cartesian4` makes sense though, it's in the name after all...